### ISSUE TITLE
Video chat should stay active as you move from room to room

### ISSUE THREAD

**Issue by lazerwalker** [lazerwalker/azure-mud#215]
This was Andy Baio's feedback. The idea is, when you leave a room, you still disconnect from all those local videochats, but we basically keep you in 'video chat mode' and automatically connect you to other videochatters when you move to a new room.

I think this ties into a broader question about the role of videochat.

**Comment by lazerwalker** [comment]
I think there is more technical and design work needed to figure out what role audio/video chat should play. But it's a reasonable technical improvement that, if you were previously connected to videochat, we auto-join you instead of making you re-join when you move to a new room.

Complexity question here: if you enter a room that doesn't have videochat, do we remember that you were previously connected the next time you enter a videochattable room? An answer to this question is just to put videochat in every room, but that's also an open design question about what role audio/video plays.

**Comment by MoyTW** [comment]
I think that, if the choice is between [a cool interaction] or [a minimap] and this we should pick the other thing, it seems like we're gonna have a reasonable number of rooms that don't have video chat anyways and I suspect that the feeling of randomly being dropped from chat depending on whether or not you wander into a different room is terrible. Especially since we're explicitly saying to turn off video chat in unconferences and use Zoom or something.

**Comment by lazerwalker** [comment]
This would be nice to have, but it's blocked on ACS and seeing if that's possible.

**Comment by lazerwalker** [comment]
Closing in favor of #361, a more nuanced approach to this
